### PR TITLE
PXP-581: [CLI] Implement the deployment section

### PR DESCRIPTION
**Pull request by jk-gan**
## Summary

> [!IMPORTANT]  
> This PR is implemented on top of #133 , so this PR is based on that branch.

Ticket: [PXP-581](https://mindvalley.atlassian.net/browse/PXP-581)

### Added
- [x] Display the deployment data
Screenshot 2023-08-21 at 2 50 32 PM

### Changed
- [x] Add padding to builds area

### Fixed
- [x] The namespace selection will now show the correct available namespaces 
Screenshot 2023-08-21 at 3 23 53 PM

- [x] Remove the `Refreshing Token ...` message which break the tui layout

[PXP-581]: https://mindvalley.atlassian.net/browse/PXP-581?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/134#" title="Author self-approved">jk-gan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/feat/tui-build-artifact/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by onimsha — APPROVED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/134#" title="Author self-approved">jk-gan</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/134#pullrequestreview-1588372205" title="Approved">onimsha</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/feat/tui-build-artifact/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
